.travis.yml

```diff
@@ -6,8 +6,16 @@ php:
   - 5.6
   - 7.0
   - 7.1
-  - nightly
-  - hhvm
+  - nightly # ignore errors, see below
+  - hhvm # ignore errors, see below
+
+# lock distro so new future defaults will not break the build
+dist: trusty
+
+matrix:
+  allow_failures:
+    - php: hhvm
+    - php: nightly
```

Comment:
The short version is that HHVM builds on Travis repeatedly broke in the past and needed updates despite otherwise working perfectly fine and nightly builds can't really be guaranteed in the future as they have the potential of breaking once PHP 8 or PHP 9 is out.

Also, this is now in line with all other components :+1: 

See also https://github.com/reactphp/socket/pull/109 and https://github.com/reactphp/dns/pull/56